config/auth_config.yml

```diff
@@ -0,0 +1,28 @@
+server:
+  addr: ':5001'
+  # TODO: not sure what it's for, the doc also have away for letsencrypt
+  certificate: '/path/to/server.pem'
```

Comment:
It's probably easier to let nginx do this, so we only manage the certs in a single app.  We'll get @manekenpix to weigh in with his advice.